### PR TITLE
Save bundle to s3

### PR DESCRIPTION
**Pull request by romulodl**
- add bundle to S3
 - compatible with old way (saving locally)
 - improve speed
 - remove files not being used

### CODE REVIEW

File: app/Admin/HooksLoader.php [hunk 1]
```diff
@@ -18,43 +12,38 @@ public function __construct( Compiler $compiler ) {
...
-
 	public function create_or_update( $post_id ) {
 		if ( wp_is_post_revision( $post_id ) ) {
 			return;
 		}
 
-		$post = new TimberPost( $post_id );
-		if ( in_array($post->post_type, [ 'post', 'sponsored_post', 'longform' ] ) === false ) {
+		$post = get_post( $post_id );
+		if ( false === in_array($post->post_type, [ 'post', 'sponsored_post', 'longform' ], true ) ) {
```
Comment (leonoragilmour) [inline app/Admin/HooksLoader.php]: YODA

File: app/Admin/HooksLoader.php [hunk 2]
```diff
@@ -18,43 +12,38 @@ public function __construct( Compiler $compiler ) {
...
 			return;
 		}
 
 		if ( $post->post_status !== 'publish' ) {
 			return;
 		}
 
-		if(!isset($post->article_should_publish_to_apple_news) || $post->article_should_publish_to_apple_news !== '1') { // for some god forsaken reason, true_false is stored as a string
+		$should_publish_to_apple_news = (bool) get_post_meta( $post_id, 'article_should_publish_to_apple_news', true );
+		if ( ! $should_publish_to_apple_news ) {
```
Comment (leonoragilmour) [inline app/Admin/HooksLoader.php]: this is nice... i nearly lost my mind here

File: app/Admin/HooksLoader.php (hunk 1 above)
Comment (romulodl) [reply app/Admin/HooksLoader.php]: yoda

Comment (romulodl) [reply app/Admin/HooksLoader.php]: Removed btw 👍

File: app/Admin/HooksLoader.php (hunk 2 above)
Comment (leonoragilmour) [reply app/Admin/HooksLoader.php]: also 👍 picking up the caption is empty stuff!

**Pull request merged**